go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 replace (
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
 	github.com/graymeta/stow => github.com/kastenhq/stow v0.2.6-kasten.1.0.20220726203146-8a90401257d4
-	github.com/kopia/kopia => github.com/kastenhq/kopia v0.0.0-20221221212937-f1fc864f1dd0
+	github.com/kopia/kopia => github.com/kastenhq/kopia v0.0.0-20230209084412-6e05ca94108a
```

Comment:
Are we switching to upstream kopia in a different PR?

Reply:
Upstream kopia doesn't have a fix for some files which were included in forked repo. Until we raise a PR or they fix it, we cannot change to upstream kopia